### ISSUE TITLE
自定义nclass

### ISSUE THREAD

**Issue by chw177** [fengshilin/tf2.0-FCN#2]
你好，

我想把class number从2改成了3，因为最后segmentation有三个大类加一个background。但是有个报错“W tensorflow/core/common_runtime/base_collective_executor.cc:217] BaseCollectiveExecutor::StartAbort Invalid argument: logits and labels must be broadcastable: logits_size=[3456000,3] labels_size=[3456000,2]”。 请问在哪里读取的label的size呢？

另外，请问cost function是在哪里define的？

谢谢
